frontend/assets/koblenz/fonts/GUST e-foundry License.txt

```diff
@@ -0,0 +1,29 @@
+This is a preliminary version (2006-09-30), barring acceptance from
```

Comment:
:upside_down_face: Does this file name has to be this way? I'd prefer it if there would be no whitespaces/special characters.

Reply:
i just copied everything from an downloaded archieve